lib/apia/request.rb

```diff
@@ -36,7 +36,7 @@ def json_body
     end
 
     def body?
-      has_header?('rack.input')
+      has_header?(::Rack::RACK_INPUT)
```

Comment:
ℹ️ not strictly necessary, but I always prefer to use constants when they're available